presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -534,6 +560,15 @@ private int findLastSegmentStart(PagesHashStrategy pagesHashStrategy, Page page)
 
     private void closeAggregationBuilder()
     {
+        partialAggregationController.ifPresent(
+                controller -> controller.onFlush(
+                        inputBytesProcessed,
+                        inputRowsProcessed,
+                        aggregationBuilder instanceof SkipAggregationBuilder ? OptionalLong.empty() : OptionalLong.of(uniqueRowsProduced)));
```

Comment:
nit: Add a comment on what an empty uniquerowsproduced means and how it differs from 0 unique rows.